### DISCUSSION TITLE
What's our governance relationship with meetup groups/organizers?

### DISCUSSION THREAD

**Discussion by jawache** [Green-Software-Foundation/discussions#12] [Ask Anything] [answered]
(Note: we're asking specific question to meetup.com, our governance relationship might just be that we pay for the meetup group or don't pay for the meetup group, but every other decision ultimately lies with the meetup organizer - checking details)

- I believe the idea we should have re: these meetup groups is that tthe organizers of a meetup group **are** the governance for that meetup group. We are simply here to nurture and support them.
- If they go in a direction that doesn't align with our values, our approach would be to ask them to remove our branding from the meetup group and to stop supporting them financially. (NOTE: We should make sure this is explicit in our trademark policy)
- This is simply a sponsorship arrangement, for instance we can sponsor other green software related meetup groups through our meetup program, it doesn't mean we **own** that meetup group, just that we believe in them and want to support them.

**Comment by jawache** [comment, marked as answer]
I discussed this in more details in this discussion https://github.com/Green-Software-Foundation/meetup/discussions/25 they are both related. 

TL;DR: 

- Meetup organizers have a mandate to run their meetup groups in whatever way they see fit as long as their actions falls under our policies.
- GSF governance should only get involved in the setting of the policies and the decisions regarding what to do if those policies are broken.
I.e. The Community WG decides the rules and ultimately decides what happens if people breaks the rules but shouldn't dictate how people should run their meetup groups, let the meetup organizers govern and manage themselves.
- The actions we can take if a policy is broken is determined by if they are a GSF Supported Meetup Groups or a GSF Sponsored Meetup Group, more details in https://github.com/Green-Software-Foundation/meetup/discussions/25

**Reply by jawache** [reply]
This was agreed in the Community WG call
